ucc/compile.py

```diff
@@ -24,13 +29,115 @@
     )
 supported_circuit_formats = ConversionGraph().nodes()
 
+# Check if `qmprs` is installed for AQC compilation
+qmprs_available = importlib.util.find_spec("qmprs") is not None
```

Comment:
Since this is only used once, I'd either inline this check to the function below, or perhaps better, import this from the aqc module so its defined in one place.